tensorflow/compiler/tf2tensorrt/convert/convert_nodes.cc

```diff
@@ -4721,6 +4721,58 @@ Status ConvertSoftmax(OpConverterParams* params) {
   return Status::OK();
 }
 
+Status ConvertLogSoftmax(OpConverterParams* params) {
```

Comment:
Instead of adding new converters to convert_nodes.cc, please prefer to:
- derive it from [OpConverterBase](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/compiler/tf2tensorrt/convert/op_converter.h)
- place it under [convert/ops](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/compiler/tf2tensorrt/convert/ops)
- Place the validation code into `Validate()`, and the actual conversion into `Convert()`, see the [tile converter](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/compiler/tf2tensorrt/convert/ops/tile.cc) as an example
- don't forget to add an entry to [BUILD](https://github.com/tensorflow/tensorflow/blob/cf29afd3e937e65cdc8d4dff824bcc0af09ab43d/tensorflow/compiler/tf2tensorrt/BUILD#L682)